travis/build_osx_package.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -ex
+set -x
```

Comment:
Don't we want to exit if one of commands in this script returns non-zero exit code ?